src/app/components/drawer/DrawerArticles.js

```diff
@@ -59,7 +59,7 @@ const DrawerArticlesComponent = ({ team }) => {
               <div className={styles.listLabel}>
                 <FormattedMessage defaultMessage="Claim & Fact-Checks" description="Label for a list displayed on the left sidebar that includes items that have claim & fact-checks" id="articlesComponent.claimAndFactChecks" tagName="span" />
               </div>
-              <ArticleCoreListCounter teamSlug={team.slug} type="fact-check" />
+              <DrawerListCounter key={team.factChecksCount} numberOfItems={team.factChecksCount} />
```

Comment:
These `key` props passed to the `DrawerListCounter` seem unnecessary as this isn't the output of a list.map and `DrawerListCounter` has no internal state that would need being reset upon key updating. Can you plz remove them?

Reply:
Removed